tests/validation/tests/v3_api/common.py

```diff
@@ -2217,6 +2217,7 @@ def generate_template_global_role(name, new_user_default=False, template=None):
 def wait_for_backup_to_active(cluster, backupname,
                               timeout=DEFAULT_TIMEOUT):
     start = time.time()
+    timeout = start + 300
```

Comment:
could we change this to 

```python
timeout = start + timeout
```

This way we use the default timeout value instead of a fixed value of 300